lib/sdf-server/src/service/change_set.rs

```diff
@@ -80,7 +78,9 @@ impl IntoResponse for ChangeSetError {
             ChangeSetError::ActionAlreadyEnqueued(_) => {
                 (StatusCode::NOT_MODIFIED, self.to_string())
             }
-            ChangeSetError::ChangeSetNotFound => (StatusCode::NOT_FOUND, self.to_string()),
+            ChangeSetError::DalChangeSet(DalChangeSetError::ChangeSetNotFound(..)) => {
```

Comment:
I considered doing an `inner.into_response()` here, but changeset not existing could be a 500 in some callsites (it's only a 404 if the changeset is one you specified in the request) so I left it specific here.